virttest/utils_conn.py

```diff
@@ -1724,7 +1798,13 @@ def conn_setup(self):
             try:
                 libvirtd_service = utils_libvirtd.Libvirtd(
                     session=client_session)
-                libvirtd_service.restart()
+                if self.libvirt_ver:
+                    libvirtd_service.stop()
+                    process.run("systemctl daemon-reload")
```

Comment:
we may need check the result for process.run().

Reply:
will raise exception if cmd failed